server/api.js

```diff
@@ -211,7 +165,34 @@ router.post("/laptop_donation", (req, res) => {
 		email,
 		deliveryOption,
 	])
-		.then(() => res.send("result.rows"))
+		.then(async () => {
+			let laptopDonationResult = await db.query(
```

Comment:
This is wrong. You already have a laptop_donation: the one you just inserted

Reply:
@gregdyke 
Done that